assets/global.js

```diff
@@ -935,6 +936,10 @@ class VariantSelects extends HTMLElement {
         if (inventoryDestination) inventoryDestination.classList.toggle('visibility-hidden', inventorySource.innerText === '');
 
         this.toggleAddButton(!this.currentVariant.available, window.variantStrings.soldOut);
+
+        const quantityFormCurrent = document.getElementById(`Quantity-Form-${sectionId}`);
+        const quantityFormUpdated = html.getElementById(`Quantity-Form-${sectionId}`);
+        if (quantityFormCurrent && quantityFormUpdated) quantityFormCurrent.innerHTML = quantityFormUpdated.innerHTML;
```

Comment:
Is there any alternative to replacing this element with a new one, such as updating values to match? 

If possible, I would love to avoid replacing elements because, especially on slower connections, users can quickly tab to the input and start working on it while the request is still being processed. Once the request is completed, the active element is replaced and, as a result, keyboard focus is lost.

https://user-images.githubusercontent.com/37168033/215180779-42c51357-948a-4cc4-a5b1-035cea0a019d.mp4

Ideally, we would replace only the values that are needed (max, min, value, etc) via JS and not completely replace the element.

Alternatively, we could block user interactions with the quantity inputs until these elements are updated.